bigchaindb_driver/driver.py

```diff
@@ -39,6 +40,12 @@ def __init__(self, *, public_key, private_key, node):
         self.public_key = public_key
         self.private_key = private_key
         self.node = node
+        self.transport = transport_class(node)
+
+    def retrieve(self, txid):
+        path = '/transactions' + '/' + txid
```

Comment:
`/transactions/` should probably be in a api_urls.py module or something. Also, don't need the extra concat and (maybe) using `.format()` is better?